src/motion/features/layout/Animate.tsx

```diff
@@ -153,9 +153,11 @@ class Animate extends React.Component<AnimateProps> {
             }
 
             if (prevParentViewportBox) {
-                isRelative = true
-                origin = calcRelativeOffset(prevParentViewportBox, origin)
-                target = calcRelativeOffset(parentLayout, target)
+                if (prevParent || (!prevParent && !(originBox || targetBox))) {
```

Comment:
Can we extract these in a well named constant that explains what we're checking for?